spec/spec_helper.rb

```diff
@@ -1,14 +1,11 @@
 # Make sure specs run with the definitions from test.rb
 ENV['ROBOT_ENVIRONMENT'] = 'test'
 
-require "codeclimate-test-reporter"
-CodeClimate::TestReporter.start
+require 'simplecov'
+SimpleCov.start
+
 require 'coveralls'
 Coveralls.wear!
-SimpleCov.formatter = SimpleCov::Formatter::MultiFormatter[
-  CodeClimate::TestReporter::Formatter,
-  Coveralls::SimpleCov::Formatter
-]
```

Comment:
Yes, I saw that but `CodeClimate` was not defined -- the latest version of codeclimate says it's all built-in now. See https://github.com/codeclimate/ruby-test-reporter/blob/master/CHANGELOG.md#v100-2016-11-03 and https://docs.codeclimate.com/docs/ruby

Honestly after looking at this for a few minutes it's not clear at all what the current practice is. Let me see if I can find an example in sul-dlss...